frontend/lib/about/language_change.dart

```diff
@@ -1,6 +1,9 @@
 import 'package:ehrenamtskarte/build_config/build_config.dart' show buildConfig;
 import 'package:ehrenamtskarte/l10n/translations.g.dart';
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../configuration/settings_model.dart';
 
 Map<String, String> languages = {'en': 'Englisch', 'de': 'Deutsch'};
```

Comment:
Those should actually also go in the translations, the language names differ in german and english for example.

Reply:
i used a  `Map` for this in translations